lib/BloqadeLattices/src/lattice.jl

```diff
@@ -291,24 +511,6 @@ function collect_atoms(mg::MaskedGrid)
     return AtomList(map(ci -> (mg.xs[ci.I[1]], mg.ys[ci.I[2]]), findall(mg.mask)))
 end
 
-# generating docstrings
```

Comment:
Does this not pass the docstring test we implemented??

Reply:
I don't remember but I ended up removing this code because I preferred each function to have its own docstring and output from Documenter over gluing things together.